packages/design-system/src/components/Button/variations/Button.destructive.tsx

```diff
@@ -8,7 +8,7 @@ const button: StyledFunction<typeof ButtonPrimary> = styled(ButtonPrimary);
 const ButtonDestructive: React.FC<ButtonProps> = button.attrs({
 	className: 'btn--destructive',
 })`
-	--t-button-background-color: ${({ theme }) => theme.colors?.buttonDestructiveBackgroundColor};
+	--t-button-background-color: orange;
```

Comment:
You madman

Reply:
I'm not afraid of you DS guys !

Reply:
Ok in fact I am afraid and decided to revert